packages/components/src/theme.ts

```diff
@@ -2,12 +2,28 @@ export type CSSObject = object;
 
 export type BaseTheme = {
```

Comment:
Should make all scales required for a theme? 🤔

Reply:
I guess we should make the values required that we actually use in the components?